src/Plugin/InvokeStrategy/HandleCommandStrategy.php

```diff
@@ -26,6 +26,10 @@ function (ActionEvent $actionEvent): void {
                 $message = $actionEvent->getParam(MessageBus::EVENT_PARAM_MESSAGE);
```

Comment:
move this code up, message and handlers don't need to get fetched before this check.